sources/packages/web/src/components/common/AwardTable.vue

```diff
@@ -55,13 +59,26 @@ export default defineComponent({
       if (awardValue === null) {
         return "-";
       }
+      if (awardValue) {
+        return (
+          "$" + awardValue.toLocaleString("en-CA", { minimumFractionDigits: 2 })
```

Comment:
Can we move this part to a `useFormatters` as it has a potential to be re-used ? 

[useFormatters](https://github.com/bcgov/SIMS/blob/main/sources/packages/web/src/composables/useFormatters.ts)

```ts
awardValue.toLocaleString("en-CA", { minimumFractionDigits: 2 })
```